pkgs/development/tools/misc/patchelf/default.nix

```diff
@@ -1,16 +1,17 @@
 { stdenv, fetchurl }:
 
 stdenv.mkDerivation rec {
-  name = "patchelf-0.9";
+  name = "patchelf-0.11";
 
   src = fetchurl {
     url = "https://nixos.org/releases/patchelf/${name}/${name}.tar.bz2";
-    sha256 = "a0f65c1ba148890e9f2f7823f4bedf7ecad5417772f64f994004f59a39014f83";
+    sha256 = "16ms3ijcihb88j3x6cl8cbvhia72afmfcphczb9cfwr0gbc22chx";
   };
 
   setupHook = [ ./setup-hook.sh ];
 
-  doCheck = false; # fails 8 out of 24 tests, problems when loading libc.so.6
+  # fails 8 out of 24 tests, problems when loading libc.so.6
+  doCheck = stdenv.name == "stdenv-linux" || stdenv.name == "stdenv-darwin";
```

Comment:
I'm just curious and it doesn't really matter: why `stdenv.name` instead of the usual `stdenv.isLinux` and `stdenv.isDarwin`?

Reply:
Testa fail on boostrapping stdenvs, this makes sure we only run tests after the final stdenv is built.

It would be better if we had stdenv.isBootstrapping.